route53_vincent_app.tf

```diff
@@ -16,6 +16,17 @@ resource "aws_route53_record" "vincent_dev_CNAME" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "vincent_dev_cert_verify_CNAME" {
```

Comment:
You need to update the Terraform documentation in the `README.md` file accordingly.  The `terraform-docs` command can do this for you.

Reply:
I have updated the readme and pushed that change.